patches/minecraft/net/minecraft/fluid/LavaFluid.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/fluid/LavaFluid.java
++++ b/net/minecraft/fluid/LavaFluid.java
+@@ -188,6 +188,8 @@
+       return 100.0F;
+    }
+ 
++   public net.minecraftforge.fluids.FluidAttributes createAttributes(Fluid fluid) { return net.minecraftforge.fluids.FluidAttributes.VANILLA_LAVA; }
```

Comment:
Should make a `@Override` before this, just to make sure

Reply:
Good catch.